lib/src/viam_sdk_impl.dart

```diff
@@ -18,9 +24,37 @@ class ViamImpl implements Viam {
   ViamMovementService? movementService;
   ViamSensorService? sensorService;
   DataService? _dataService;
+  late AppClient _appClient;
+  late DataClient _dataClient;
+
+  ViamImpl();
```

Comment:
```suggestion
  ViamImpl._();
```

can we make the default constructor private?

Reply:
We can't -- the (newly) deprecated version needs a public constructor and I don't want to mess with anything DoR might be using